person/models.py

```diff
@@ -78,6 +78,20 @@ class GodFather(Person):
     ('UNA', 'Usuario no asociado')
 )
 
+USER_TYPE = (
+    ('SCC','Socios ASEM con cuota de socio'),
+    ('UCC','Usuarios con cuota de socio'),
+    ('UCS','Usuarios sinn cuota de socio')
+)
+
+CORRESPONDENCE = (
+    ('E', 'Email'),
+    ('CC', 'Carta con logo'),
+    ('CS', 'Carta sin logo'),
+    ('SR', 'Solo revista'),
+    ('N', 'Ninguna')
+)
+
```

Comment:
As suggested, just put all those listed above with the others, although I suppose that it is still something subjective. I know that there are others listed that I think you have not put or I do not remember now, but if you change them, then change them all if you can.

Reply:
Ok I think that makes sense, I'll change it.